crates/re_data_ui/src/image.rs

```diff
@@ -12,18 +12,17 @@ use re_viewer_context::{
     ViewerContext,
 };
 
-use crate::image_meaning_for_entity;
+use crate::{image_meaning_for_entity, label_for_ui_layout};
 
 use super::EntityDataUi;
 
 pub fn format_tensor_shape_single_line(shape: &[TensorDimension]) -> String {
     const MAX_SHOWN: usize = 4; // should be enough for width/height/depth and then some!
     let shapes = shape.iter().take(MAX_SHOWN).join(", ");
-    if shape.len() > MAX_SHOWN {
-        format!("{shapes}…")
-    } else {
-        shapes
-    }
+    format!(
+        "shape: {shapes}{}",
```

Comment:
the `shape:` part isn't useful and just takes space imho. Interestingly, on the screenshots you took one has it and one doesn't ;-)
(I checked locally, it's always there now)

Reply:
I figured out a decent way, see updated screenshots. I also added some cases to the checklist.